lib/project_types/extension/features/argo.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+require 'base64'
+require 'shopify_cli'
+require 'semantic/semantic'
+
+module Extension
+  module Features
+    class Argo
+      include SmartProperties
+
+      property! :git_template, converts: :to_str
+      property! :renderer_package_name, converts: :to_str
```

Comment:
`git_template` and `renderer_package_name` are now injectable. The rest of the file is marked as new do to renaming and git not properly picking up on the file location change.